modules/aws-backup-source/resources/post_build_version/post_build_version.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+import urllib.request
+import urllib.parse
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+
+API_ENDPOINT = os.environ.get('API_ENDPOINT')
+MODULE_VERSION = os.environ.get('MODULE_VERSION')
+AWS_ACCOUNT_ID = os.environ.get('AWS_ACCOUNT_ID')
+API_TOKEN = os.environ.get('API_TOKEN')
+
+
+def lambda_handler(event, context):
+    """
+    Handles events from AWS EventBridge (triggered by AWS Backup job completion).
+    It sends the module version information via HTTP POST to a configured API endpoint.
+
+    Args:
+        event (dict): The EventBridge event data, including AWS Backup job details.
+        context (object): Lambda context runtime information.
+    """
+    logger.info(f"Received Event: {json.dumps(event)}")
+
+    if not API_ENDPOINT or not MODULE_VERSION:
+        logger.error("Configuration error: API_ENDPOINT or MODULE_VERSION environment variables are missing.")
+        return {
+            'statusCode': 500,
+            'body': json.dumps({'message': 'Configuration error.'})
+        }
+
+    payload = {
+        "version": MODULE_VERSION,
+        "awsAccount": AWS_ACCOUNT_ID
+    }
```

Comment:
It's worth us having a payload version as well as a module version, which would make this look like:


```suggestion
    payload = {
        "version": "v1",        
        "moduleVersion": MODULE_VERSION,
        "awsAccount": AWS_ACCOUNT_ID
    }
```

I'm thinking make `version` unambiguously a string so it can never be mishandled.